onnxruntime/core/graph/graph_viewer.cc

```diff
@@ -52,6 +55,10 @@ bool GraphViewer::GetInitializedTensor(const std::string& tensor_name, const ONN
   return graph_->GetInitializedTensor(tensor_name, value);
 }
 
+bool GraphViewer::CanOverrideInitializer() const noexcept {
+  return graph_->CanOverrideInitializer();
```

Comment:
Would it be more consistent to add support for GraphViewer to graph_utils?

Reply:
I thought about it.
It might require to expose more Graph's member functions in GraphViewer.
No sure whether that is what we really want.

Reply:
What's the use case for this method? Everywhere else doing this check is early on during optimization where the Graph instance is available, so wondering where this call needs to happen and why only the GraphViewer is available.